src/cdk/rxjs/rxjs.md

```diff
@@ -0,0 +1,36 @@
+### RxJS Usage
+When dealing with RxJS operators, it is best to import the operator functions directly (e.g.
+`import "rxjs/operator/map"`), as opposed to using the "patch" imports which pollute the global Observable object (e.g. `import "rxjs/add/operator/map"`):
+
+```ts
+// NO
+import 'rxjs/add/operator/map';
+someObservable.map(...).subscribe(...);
+
+// YES
+import {map} from 'rxjs/operator/map';
+map.call(someObservable, ...).subscribe(...);
+```
+
+#### RxChain
+Because this approach can be inflexible when dealing with long chains of operators. You should use the `RxChain` class to help with it:
+
+```ts
+// Before
+someObservable.filter(...).map(...).do(...);
+
+// After
+RxChain.from(someObservable).call(filter, ...).call(map, ...).subscribe(...);
+```
+
+#### Available Operators
+Many operators are provided in via the `RxChain`, these operators include:
```

Comment:
I don't think that we should include the list operators here, because it means that we'll have to maintain it.

Reply:
Yeah, you're right.  I went back and forth on it before I put it in.